### ISSUE TITLE
Cannot select menu items on iOS with VoiceOver since 5.8.0

### ISSUE THREAD

**Issue by mbellagamba** [JedWatson/react-select#5848]
Using VoiceOver on iOS is not possible to select options. Issue #5190 already pointed out that is not possible to select an option with the canonical double tab. However users could still rely on a the workaround suggested in [that comment](https://github.com/JedWatson/react-select/issues/5190#issuecomment-1164462937). In release 5.8.0 it is not even possible to scroll through the options because the listbox closes after the first slide forward.

To reproduce the issue open https://2qmg83.csb.app/ using Safari on iOS with VoiceOver.
1. Open the select double tapping on it.
2. Swipe forward (from right to left)
I expect the focus moves to the first option, instead the listbox closes.

The URL to reproduce the issue is generated from the [CodeSandbox](https://codesandbox.io/p/sandbox/codesandboxer-example-2qmg83) used in the [react-select.com](https://react-select.com) homepegage.

**Comment by doubleforte** [comment]
Any progress on this? It's very broken for VoiceOver on iOS. :(
